src/core/packages/saved-objects/api-server-internal/src/lib/apis/bulk_create.ts

```diff
@@ -53,7 +53,9 @@ type ExpectedResult = Either<
       id: string;
       accessControl?: SavedObjectAccessControl;
     };
-    preflightCheckIndex?: number;
+    id: string;
+    type: string;
```

Comment:
I am not 100% happy about this change, but it was the least intrusive way for making this type compatible with the `filterInaccessibleObjectsForBulkAction` function.